begin/cmdline.py

```diff
@@ -37,10 +37,11 @@ def __init__(self, env_prefix=None, config_file=None, config_section=None):
         self._use_env = env_prefix is not None
         self._prefix = '' if not self._use_env else env_prefix
         self._parser = configparser.ConfigParser()
-        self._section = config_section
+        self._section = None
```

Comment:
This is rather confusing (having both a `section`, `_section` and a property with this name.
Let's try to find a clearer name for this? Can we maybe delete some of these?